src/whylogs/app/dataset.py

```diff
@@ -0,0 +1,81 @@
+import os
+from typing import Any, Callable, List, Optional, Tuple
+import ABC
+
+from PIL import
```

Comment:
This import creates an explicit dependency on PIL. Basically nobody can use this class unless they have PIL installed